pyisic/_standards/gced2011/gced2011_to_nace2.py

```diff
@@ -0,0 +1,1008 @@
+"""`Concordance between GCED2011 and NACE2 <http://www.stat.kg/stat.files/class.files/%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%20%D0%93%D0%9A%D0%AD%D0%94-3.pdf>_`.
+
+All GCED2011 codes that are 4 digits long map directly to the same NACE2 code.
```

Comment:
Is this really true with concordances like L23? That seems to be 3 digits. Can we change the language or confirm if that mapping is valid?

Reply:
whoops, I meant 4 or less